statsmodels/iolib/foreign.py

```diff
@@ -956,6 +967,11 @@ def genfromdta(fname, missing_flt=-999., encoding=None, pandas=False,
         If convert_dates is True, then Stata formatted dates will be converted
         to datetime types according to the variable's format.
     """
+    warnings.warn(
+        "genfromdta is deprecated as of 0.10.0 and will be removed in a "
+        "future version.  Use pandas.read_stata instead.",
```

Comment:
Same here.

Reply:
The other ones I'm clear on.  What are you saying should be added here?

Reply:
This is OK since it is the function interface.